### PR TITLE
Fix #4216: madx to elegant import

### LINKED ISSUE #4216: madx import fails: >=' not supported between instances of 'str' and 'int'
Importing a MADX file to elegant that was exported from https://www.sirepo.com/opal#/visualization/3CeErNY9 gives an error:
`'>=' not supported between instances of 'str' and 'int'`

[CM_test.txt](https://github.com/radiasoft/sirepo/files/8201348/CM_test.txt)
`

### PR DESCRIPTION
**Pull request by gurhar1133**
Opal now exports working madx
elegant to madx conversions now work (and madx to elegant)

### CODE REVIEW

**Comment by gurhar1133** [comment]
Changes made

File: sirepo/template/elegant.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
+import ast
```
Comment (moellep) [inline sirepo/template/elegant.py:8]: I think this import can be removed as well as the ast related imports in template/opal

File: tests/template/madx_converter_test.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
+from importlib.resources import read_text
```
Comment (moellep) [inline tests/template/madx_converter_test.py:8]: Another unused import

File: sirepo/template/madx_converter.py [hunk 3]
```diff
@@ -43,7 +68,8 @@ def to_madx(self, data):
         self.__init_direction(data, self.sim_type, madx.SIM_TYPE)
         self.field_map = self.full_field_map.to_madx
         self.drift_type = self.full_field_map.from_madx.DRIFT[0]
-        return self._convert(data)
+        d = self._convert(data)
+        return d
```
Comment (moellep) [inline sirepo/template/madx_converter.py:72]: Can remove temporary variable

**Review by moellep — COMMENTED** [review]
Looks great - just a few minor import issues.

**Comment by gurhar1133** [comment]
Made those changes and all tests passed

**Pull request merged**